src/lib/Microsoft.Health.Fhir.Ingest/Host/DeviceDataNormalizationExtensions.cs

```diff
@@ -46,6 +47,8 @@ public static IServiceCollection AddNormalizationExceptionTelemetryProcessor(thi
             EnsureArg.IsNotNull(serviceCollection, nameof(serviceCollection));
             EnsureArg.IsNotNull(configuration, nameof(configuration));
 
+            serviceCollection.AddSingleton<IErrorMessageService, ErrorMessageService>();

```

Comment:
ErrorMessageService is causing a compile time error for me. It looks like it is not included in this branch.

Reply:
yep, my bad. I needed to remove this line of code for OSS. Should be fixed now.